mavros/src/plugins/waypoint.cpp

```diff
@@ -543,6 +818,13 @@ class WaypointPlugin : public plugin::PluginBase {
 			else {
 				enable_partial_push = m_uas->is_ardupilotmega();
 			}
+			if (wp_nh.hasParam("use_mission_item_int")) {
+				wp_nh.getParam("use_mission_item_int", use_mission_item_int);
+			}
+			else {
+				use_mission_item_int = m_uas->get_capabilities() & MAV_PROTOCOL_CAPABILITY_MISSION_INT;
+				mission_item_int_support_confirmed = use_mission_item_int;
```

Comment:
As i remember, connection_cb() called before sys_status starts doing capabilities negotiation.

Reply:
This seems to be correct, I have not fixed this yet in the latest commits but is there a way that I can register a cb to the capabilities information being available or should I just do check in one of the actions that would execute earliest?

Reply:
As i remember there no such event in UAS yet. So i think you should add event processor to the UAS, then update base class to have `capabilities_cb()` and methods to support it.

That would be useful for other plugins too.